metadata-ingestion/README.md

```diff
@@ -322,6 +323,7 @@ source:
     password: pass
     host_port: localhost:5432
     database: DemoDatabase
+    database_identifer: DatabaseNameToBeIngested
```

Comment:
Name this field as `database_alias` ?

Reply:
Good suggestion. That is more appropriate. Done.